### PR TITLE
feat: add sso command bot sample

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: sso-command-bot/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# TeamsFx files
+node_modules
+.fx/configs/localSettings.json
+.fx/states/*.userdata
```
Comment (SLdragon) [inline sso-command-bot/.gitignore]: Seems redundant with `.fx/states/*`

File: sso-command-bot/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            "type": "pwa-chrome",
+            "request": "launch",
+            "url": "https://teams.microsoft.com/l/app/${localTeamsAppId}?installAppPackage=true&webjoin=true&${account-hint}",
+            "cascadeTerminateToConfigurations": [
+                "Attach to Bot"
+            ],
+            "presentation": {
+                "group": "all",
+                "hidden": true
+            }
```
Comment (wenytang-ms) [inline sso-command-bot/.vscode/launch.json:48]: better to add "internalConsoleOptions": "neverOpen"
according to https://github.com/OfficeDev/TeamsFx-Samples/pull/484

Comment (SLdragon) [reply sso-command-bot/.vscode/launch.json:48]: Thanks, will update~

Comment (SLdragon) [reply sso-command-bot/.vscode/launch.json]: Updated

**Review by KennethBWSong — APPROVED** [review]

File: command-bot-with-sso/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## Edit the manifest
+
+You can find the Teams manifest in `templates/appPackage/manifest.template.json`. It contains template arguments with `{...}` statements which will be replaced at build time. You may add any extra properties or permissions you require to this file. See the [schema reference](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema) for more.
+
+## Edit Azure AD manifest
+
+You can find the Azure AD manifest in `templates/appPackage/aad.template.json`, and this file can help you manage Azure AD app for SSO. Follow this [document](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/aad-manifest-customization) to update Azure AD manifest if you want to customize AD app.
+
+## Deploy to Azure
```
Comment (wenytang-ms) [inline command-bot-with-sso/README.md:43]: if we need code conduct?
https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/share-now

Comment (SLdragon) [reply command-bot-with-sso/README.md:43]: OK, thanks for reminding, will update

Comment (SLdragon) [reply command-bot-with-sso/README.md:43]: Updated

**Review by wenytang-ms — APPROVED** [review]

**Pull request merged**